libvast_test/src/table_slices.cpp

```diff
@@ -60,23 +59,7 @@ table_slices::table_slices() {
     {"vj", vector_type{subnet_type{}}},
     {"vk", vector_type{port_type{}}},
     // {"vl", vector_type{vector_type{count_type{}}}},
-    // {"vm", vector_type{set_type{bool_type{}}}},
     // {"vn", vector_type{map_type{count_type{}, bool_type{}}}},
```

Comment:
Can you `s/n/m` for all entries here so the it's easier to keep track?

Reply:
I'm not sure what you mean here. Could what is supposed to be replaced? The field name?

Reply:
The variables were named quite consistently, counting up from a to n for the different types. Now that m is gone, n should be renamed to m for all type combinations.

Reply:
Got it; changed.